mlflow/pipelines/utils/metrics.py

```diff
@@ -139,5 +139,17 @@ def _load_custom_metric_functions(
         ) from e
 
 
+def _load_one_custom_metric_function(pipeline_root: str, metric: PipelineMetric) -> callable:
+    try:
+        sys.path.append(pipeline_root)
+        custom_metrics_mod = importlib.import_module("steps.custom_metrics")
+        return getattr(custom_metrics_mod, metric.custom_function)
+    except Exception as e:
+        raise MlflowException(
+            message="Failed to load custom metric functions",
+            error_code=BAD_REQUEST,
+        ) from e
+
+
```

Comment:
Can reuse the existing `_load_custom_metric_functions` by just specifying a single metric in the `metrics` list?

Reply:
Great point. Done!